docs/step-by-step/kubeapps-on-tkg/step-3.md

```diff
@@ -0,0 +1,125 @@
+
+  ![App repositories modal part 2](./img/apprepository-detail-types.png)
+
+- The _Associate Docker Registry_ field specifies an `imagePullSecret` for images that are to be pulled from the private registry. It is possible to create a fresh secret or choose an existing one.
+
+  ![App repositories modal part 3](./img/apprepository-detail-credentials.png)
+
+- The _Custom CA Certificate_ field specifies a CA certificate to use (with an option to skip the TLS verification if required).
+- The _Custom Sync Job_ field specifies the synchronization template to use when periodically pulling the latest changes from the application repository.
+
+  ![App repositories modal part 4](./img/apprepository-detail-advanced.png)
+
+6. Click the _Install Repo_ button to finish the process and add the repository to Kubeapps.
+
+Retrieving the catalog from the repository will take a few minutes. Once complete, the contents of the repository will become available in Kubeapps. To check the progress of the synchronization task, use the following command, replacing REPOSITORY-NAME with the name configured above:
+
+```bash
+kubectl logs -n kubeapps -l apprepositories.kubeapps.com/repo-name=REPOSITORY-NAME
+```
+
+The following sections demonstrate the process of filling the _Add An App Repository_ form for two specific examples: the VMware Marketplace™ and the VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced.
+
+### Add the VMware Marketplace™
+
+> **NOTE**: This repository is currently under heavy development. Therefore, the URL used below is subject to change.
+
+The public content from the VMware Marketplace™ repository can be retrieved at `https://charts.market.csp.vmware.com/bitnami`. Since this is a public repository, it is only necessary to configure the following values in the _Add An App Repository_ form:
+
+- _Name_: Add a descriptive name, such as `vmware-marketplace`.
+- _URL_: Use the endpoint URL `https://charts.market.csp.vmware.com/bitnami`.
+
+![Adding the public repository](./img/apprepository-example-marketplace.png)
+
+Once complete, the public catalog will be visible in Kubeapps, as shown below:
+
+![Catalog after adding the public repository](./img/kubeapps-catalog-marketplace.png)
+
+### Add the VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced
+
+The private content from the VMware Tanzu™ Application Catalog™ for Tanzu™ Advanced repository can be retrieved at `https://registry.pivotal.io/tac-for-tanzu-advanced/charts/`. Since this is a private OCI registry, there are two key differences to note in the process:
+
+- It is necessary to specify repository authentication credentials using the `_Basic Auth` option and defining the correct username and password. If you do not have these credentials, please reach out to your [VMware sales representative](https://www.vmware.com/company/contact_sales.html).
```

Comment:
Is the underscore intended for `_Basic Auth` here? (Didn't check the UI, but seems odd). EDIT: right, below you mention it without the underscore.